specification/resources/resource-manager/Microsoft.Management/preview/2018-01-01-preview/management.json

```diff
@@ -0,0 +1,1278 @@
+{
+  "swagger": "2.0",
+  "host": "management.azure.com",
+  "info": {
+    "version": "2018-01-01-preview",
+    "title": "Management Groups API",
```

Comment:
Are you adding this to the markdown file?

Reply:
hmm?

Reply:
The config file

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/2345/commits/e2cba494ebe7f9166f8381ecc853b09764064028